Health App/Home/HealthConcernsListVC.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  HealthConcernsListVC.swift
+//  Health App
+//
+//  Created by Koushik Dutta on 07/10/17.
+//  Copyright © 2017 Virtual Elf LLC. All rights reserved.
+//
+
+import UIKit
+import DataLayer
+
+class HealthConcernsListVC: UIViewController {
+    
+    var inControlStatus: [HealthConcern] = []
+    var notInControlStatus: [HealthConcern] = []
+    var resolvedStatus: [HealthConcern] = []
+    @IBOutlet weak var healthConcernsTableView: UITableView!
+    @IBOutlet weak var addNewHealthConcernBarButton: UIBarButtonItem!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        fetchBasedOnStatus()
+        healthConcernsTableView.reloadData()
+    }
+    
+    func fetchBasedOnStatus(){
+        inControlStatus = DatabaseManager.shared.fetchHealthConcernsBasedOnStatus(status: "In Control")
```

Comment:
use enums

Reply:
Don't use "In Control" string anywhere
only in one place all throughout the code